tools/query_verification/src/main/java/com/google/bigquery/QVQuery.java

```diff
@@ -0,0 +1,21 @@
+package com.google.bigquery;
+
+public class QVQuery {
```

Comment:
Rather than give these a QV prefix can we change this to be a custom package called verification or query.verification?